code/modules/client/preference_setup/vore/07_traits.dm

```diff
@@ -135,6 +161,11 @@
 			var/datum/trait/trait = negative_traits[T]
 			. += "<li>- <a href='?src=\ref[src];clicked_neg_trait=[T]'>[trait.name] ([trait.cost])</a></li>"
 		. += "</ul>"
+	else if (!(pref.species == SPECIES_XENOCHIMERA))
+		. += "<b>Enable Trash Can:</b> <a [pref.trait_trash_eater ? "class='linkOn'" : ""] href='?src=\ref[src];toggle_trait_trash_eater=0'><b>[pref.trait_trash_eater ? "Yes" : "No"]</b></a><br>"
+		. += "<b>Enable Brutal Predation:</b> <a [pref.trait_brutal_predation ? "class='linkOn'" : ""] href='?src=\ref[src];toggle_trait_brutal_predation=0'><b>[pref.trait_brutal_predation ? "Yes" : "No"]</b></a><br>"
+		. += "<b>Enable Bloodsucker:</b> <a [pref.trait_bloodsucker ? "class='linkOn'" : ""] href='?src=\ref[src];toggle_trait_bloodsucker=0'><b>[pref.trait_bloodsucker ? "Yes" : "No"]</b></a><br>"
```

Comment:
Why should humans for instance suddenly be able to suck blood? Absolutely not.

Reply:
While I will accept the closure, I will say. This type of logic for some reason does not apply to what we have right now? Where people do get custom species... But just recreate species they want as closely as possible, give it same name as default species, and just add scene traits to only use in scenes since its only place they can practically be used in anyway? I just do not really see why playing default species should lock you out of specific scene setups (and primarily this PR was aimed at the Trash Can specifically, being one of the best traits for scene setups and teases, while being completely locked away for anyone playing any default species, other than prommies which are niche and vox). And on the flipside, so many species have the hardvore trait that they are never going to use anyway.